doc/tech/glossary.md

```diff
@@ -146,15 +147,30 @@ Terms not defined by Kubernetes:
 - Observability = Observability is a measure of how well internal states of a system can be inferred from knowledge of its external outputs. ([Wikipedia:Observability](https://en.wikipedia.org/wiki/Observability))
 
 <a name="1.7"></a>
-## 1.7 Other Referenced Terminology
+## 1.7 Test Related Terminology
+
+- **Calibration**: The process of checking and/or adjusting a stimulus generation or measurement device with a known reference value, to improve the overall quality of the measured results.  Calibration may be very simple, such as a comparison of the configured traffic generator sending rate and measured rate using a simple SUT (System Under Test)  such as loop-back cable between interfaces, such that that the known reference value is the published nominal interface rate.
+
+- **Reference Value**: A measured or established result or outcome for comparison with new measurements. For example, the reference value or expected outcome of a Functional Test is "PASS". The reference value or expected outcome of a Performance Measurement or Benchmarking test may be the value measured for the previous SUT release, or the published value or theoretical limit of a simple SUT.
+
+- **API Testing**: Testing against a protocol specification for conformance.
+
+- **Functional Testing**: The main objective of functional testing is the verification of a specific stimulus / response within the SUT, including the expected behavior.  These tests generally result in a binary outcome, i.e. pass / fail.  For example, verification of an "API call" and its associated response, such as the instantiation of a VM (or container) and verification of the VM's (or container's) existence (expected behavior), or the ability to activate a specific feature of the SUT (e.g. SR-IOV).
```

Comment:
"The main objective of functional testing is the verification of a specific stimulus / response within the SUT, including the expected behavior." doesn't feel specific enough for "functional testing" - meaning the same could be said of non-functional testing.  e.g. non-functional testing of scaling is the verification of a specific stimulus / response within the SUT (increased load/traffic), including the expected behaviour (system auto-scaled).  

I wonder if the first sentence could be made more specific, e.g.
```suggestion
- **Functional Testing**: The main objective of functional testing is the verification of compliance against specific functional requirements using a specific stimulus / response within the SUT, including the expected behavior.  These tests generally result in a binary outcome, i.e. pass / fail.  For example, verification of an "API call" and its associated response, such as the instantiation of a VM (or container) and verification of the VM's (or container's) existence (expected behavior), or the ability to activate a specific feature of the SUT (e.g. SR-IOV).
```